pyproject.toml

```diff
@@ -1,12 +1,13 @@
 [tool.poetry]
 name = "pytest-mock-resources"
-version = "1.2.2"
+version = "1.2.3"
 description = "A pytest plugin for easily instantiating reproducible mock resources."
 authors = [
     "Omar Khan <oakhan3@gmail.com>",
     "Dan Cardin <ddcardin@gmail.com>",
     "Gabriel Michael <gabriel.j.michael@gmail.com>",
     "Prateek Pisat <pisatprateek12@gmail.com>",
+    "Lewis Moronta <lewis.moronta@gmail.com>"
```

Comment:
Too soon?

Reply:
I personally would almost rather not even have authors listed here, and instead have a CONTRIBUTORS file, since this gets included in the distribution, and people/emails aren't static...but i dont mind either way